SCons/Tool/msvs.xml

```diff
@@ -41,7 +41,7 @@ This file is processed by the bin/SConsDoc.py module
   <builder name="MSVSProject">
     <summary>
       <para>
-        Build a Microsoft Visual C++ project file and solution file.
+        Build a C++ project file and solution file.
```

Comment:
Curious, why removed The Microsoft Visual c++ project file, it's not really a c++ project file.

Reply:
It's a "C++ project" in the same way that it would be a "Visual Basic project" (`.vbproj`) or a "C# project" (`.csproj`) etc. I was trying to follow terminology I've seen...

Reply:
but it's not a generic c++ project file, its' only for visual studio..

Reply:
if you feel strongly, it's trivial to change. I was only following some things I read in other docs - perhaps those were assuming "you're already in this world, so let's not repeat ourselves".

Reply:
Just don't want someone silly to think.. oh this'll work in clion.. or visual code, or xcode..

Reply:
"Someone will just make a better idiot" and all that.  VS Code can deal with `.vcxproj` files; clion can't, but... you can first tell Visual Studio to transform it to a cmake project, and then it can.  Details you probably didn't even want to know.  xcode...  no clue.

I'll update presently.